ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -283,35 +283,69 @@ FAN_SS_DRSM8X:
   - DRSM8X
 
 FAN_EDPM_ZHM_ZTM_LPS:
-  id: "16288085548559433728"
   description: "Exhaust fan with sensors (temperature, humidity and low pressure)."
   is_canonical: true
   implements:
   - FAN
   - EDPM
   - ZHM
   - ZTM
-  uses:
-  - run_command
-  - run_status
+  opt_uses:
+  - exhaust_fan_run_command
+  - exhaust_fan_run_status
   - control_mode
   - low_differential_pressure_alarm
-  opt_uses:
   - lost_power_alarm
 
-FAN_LPS:
-  id: "16288085548559433728"
+
+FAN_FCS:
   description: "Exhaust fan with sensors (temperature, humidity and low pressure)."
   is_canonical: true
   implements:
   - FAN
-  uses:
-  - run_command
-  - run_status
+  opt_uses:
+  - lost_power_alarm
   - control_mode
   - low_differential_pressure_alarm
+  - schedule_run_command
+  - exhaust_fan_run_command
+  - exhaust_fan_run_status
+
+FAN_SA:
+  description: "Supply fan with fan speed control"
+  is_canonical: true
+  implements:
+  - FAN
+  - SS
+  uses:
+  - supply_fan_speed_percentage_command
   opt_uses:
   - lost_power_alarm
+  - control_mode
+
+
+FAN_BRS:
+  description: "Basic return fan start/stop only"
+  is_canonical: true
+  implements:
+  - FAN
+  - SS
+  opt_uses:
+  - lost_power_alarm
+  - control_mode
+  - schedule_run_command
+
+FAN_EFVSFC:
```

Comment:
Should not include exhaust fan as part of the fields. It will be clear from the entity that the type is applied to what its context is. You can use `FAN_SS_VSC` for this type.

Reply:
changed to FAN_FVSFC. exhaust_fan prefix also removed.